python/examples/handler.py

```diff
@@ -45,10 +39,7 @@ def say_ok(request, response, proxy):
     yield print_arg(request, 1)
     yield print_arg(request, 2)
 
-    response.argstreams = [
-        InMemStream(),
-        InMemStream(),
-        InMemStream("world")]
+    response.set_body(InMemStream("world"))
```

Comment:
We should get these methods to accept strings/bytes and automatically convert them to in-memory streams.

Consider using `maybe_stream` in `tchannel.tornado.peer` to do that. (Probably move it to `tchannel.tornado.stream` while you're at it.)